devito/finite_differences/finite_difference.py

```diff
@@ -200,30 +203,33 @@ def generic_derivative(expr, dim, fd_order, deriv_order, matvec=direct, x0=None,
     side = None
     # First order derivative with 2nd order FD is highly non-recommended so taking
```

Comment:
If you're changing this file, then you might as well change this line to "strongly discouraged"

Reply:
"strongly discouraged. Taking the..."